test/evergreen/find_cmake.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Adapted 'find_cmake' from mongo-c-driver evergreen infrastructure:
+#   https://github.com/mongodb/mongo-c-driver/blob/master/.evergreen/find-cmake.sh
+find_cmake ()
+{
+  if [ ! -z "$CMAKE" ]; then
+    return 0
+  elif [ -f "/Applications/Cmake.app/Contents/bin/cmake" ]; then
+    CMAKE="/Applications/Cmake.app/Contents/bin/cmake"
+    CTEST="/Applications/Cmake.app/Contents/bin/ctest"
+  elif [ -f "/opt/cmake/bin/cmake" ]; then
+    CMAKE="/opt/cmake/bin/cmake"
+    CTEST="/opt/cmake/bin/ctest"
+  elif command -v cmake 2>/dev/null; then
+     CMAKE=cmake
+     CTEST=ctest
+  elif uname -a | grep -iq 'x86_64 GNU/Linux'; then
+     if [ -f "$(pwd)/cmake-3.11.0/bin/cmake" ]; then
+      CMAKE="$(pwd)/cmake-3.11.0/bin/cmake"
+      CTEST="$(pwd)/cmake-3.11.0/bin/ctest"
```

Comment:
I'm not sure what versions of cmake are currently installed on various Linux Evergreen distro's we use. Wondering what's the reason behind checking 3.11.0 path here? Though if cmake is installed it should be captured by the previous `elif`.

Reply:
Ah I think I see, the following few lines will download 3.11.0 so it makes sense to check the path existence before downloading.

Reply:
Yep! Its meant point to the downloaded binaries, to save re-downloading.